tests/lib/rules/indent.js

```diff
@@ -829,7 +842,7 @@ ruleTester.run("indent", rule, {
                 [189, 2, 0, "VariableDeclaration"],
                 [193, 6, 4, "ExpressionStatement"],
                 [195, 6, 8, "ExpressionStatement"],
-                [197, 2, 0, "VariableDeclaration"],
+                [196, 2, 4, "VariableDeclaration"],
```

Comment:
Little afraid about this change but I can live with as we have not handled multiline yet.